packages/fxa-settings/src/lib/metrics-flow.test.ts

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import exp from 'constants';
+import { init } from './metrics-flow';
+
+const flowArg = {
+  flowId: '9999',
+  flowBeginTime: `${Date.now()}`,
+  deviceId: 'psx',
+};
+const flowQueryParam = { flowId: '8888', flowBeginTime: `${Date.now()}` };
+const flowDataAttrParam = {
+  flowId: '7777',
+  flowBeginTime: `${Date.now()}`,
+};
+
+describe('MetricsFlow', () => {
+  describe('init', () => {
+    it.skip('should initialize the metricsFlow model with body data attribues', () => {
```

Comment:
does this need to be skipped?

Reply:
No, I was debugging something but then forgot to undo.  Thanks for catching that.